src/core/client/count/injectJSONPCallback.ts

```diff
@@ -57,35 +58,34 @@ function createCountElementEnhancer({
   // Update the innerHTML which contains the count and new value..
   element.innerHTML = html;
 
-  if (storyID) {
-    const previousCount = getPreviousCount(storyID);
-    if (previousCount !== null && previousCount < currentCount) {
```

Comment:
Should we keep the `previousCount < currentCount` part of the check here? During my manual testing of this, I saw something funky like `-1 New` show up. Or maybe we check that `newCount` isn't somehow less than 0 and make it 0 if it is?

Reply:
I haven't touched this in a while, but it looks like I re-added the `previousCount < currentCount` last time I was here.

Reply:
I added a new check that only computes the diff if the `previousCount < currentCount`, otherwise the count is 0. This should allow us to always show the count as well as ensure it doesn't return a negative value similar to how it used to operate.